### PR TITLE
format the code

### PR DESCRIPTION
**Pull request by patriknw**
- just a clean test:compile

### CODE REVIEW

File: akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala [hunk 1]
```diff
@@ -33,8 +33,7 @@ object TwitterStreamQuickstartDocSpec {
 
   // format: OFF
   //#tweet-source
-  val tweets: Source[Tweet, NotUsed]
-  //#tweet-source
+  val tweets: Source[Tweet, NotUsed] //#tweet-source
```
Comment (patriknw) [inline akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala]: is this correct? otherwise we must fix this once for all

Comment (drewhk) [inline akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala]: no, this change is not correct IIRC.

File: akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala [hunk 2]
```diff
@@ -31,12 +31,13 @@ object TwitterStreamQuickstartDocSpec {
...
-  //#tweet-source
-  val tweets: Source[Tweet, NotUsed]
-  //#tweet-source
-  // format: ON
-  = Source(
+  abstract class TweetSourceDecl {
+    //#tweet-source
+    val tweets: Source[Tweet, NotUsed]
+    //#tweet-source
+  }
```
Comment (patriknw) [inline akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala]: fixed the `tweets`

**Comment by johanandren** [comment]
LGTM

File: akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala (hunk 2 above)
Comment (ktoso) [inline akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala]: Great, thanks a lot.

**Pull request merged**